File: elix/1.ex
defmodule First do
  {n, _} = Integer.parse IO.gets("")
  n = n - 1
  m = div n, 3
  m2 = div n, 5
  sum = Enum.sum(Enum.map(1..m, fn x -> x * 3 end)) + Enum.sum(Enum.map(Enum.filter(1..m2, fn x -> (rem x, 3) !== 0 end), fn x -> x * 5 end))
  IO.puts(sum)
end

File: cloj/1.clj
(def n (- (Integer. (read-line)) 1))
(def m (Integer. (quot n 3)))
(def m2 (Integer. (quot n 5)))
(def sum (reduce + (concat (map (fn [x] (* x 3)) (range 1 m)) (map (fn [x] (* x 5)) (filter (fn [x] (not= 0 (rem x 3))) (range 1 m2))))))
(println sum)
